src/main/java/frc/robot/Commands/DropPlacers.java

```diff
@@ -0,0 +1,16 @@
+package frc.robot.Commands;
```

Comment:
that's just confusing and unnecessary,
please remove

Reply:
Highly disagree. You want each command to have a single purpose.

Reply:
you should K.I.S.S, making a class instead of writing 3 lines of code is redundant.